app/components/authorization-code/login-custom/login-custom.controller.js

```diff
@@ -27,6 +27,11 @@ class LoginCustomController {
         responseType: 'code',
         scopes: ['openid', 'email', 'profile'],
       },
+      i18n: {
+        en: {
+          'primaryauth.title': 'Use john/Asdf1234 for the mock Okta server',
```

Comment:
Not sure if i18n string is being used in the view. I might be missing something. Can you double check?